test/e2e_node/remote/node_e2e.go

```diff
@@ -31,7 +31,7 @@ import (
 )
 
 const (
-	systemSpecPath = "k8s.io/kubernetes/cmd/kubeadm/app/util/system/specs"
+	systemSpecPath = "cmd/kubeadm/app/util/system/specs"
```

Comment:
It would be better to merge them into a public constant.

Reply:
Yeah, that makes perfect sense. The path is now exposed by the `k8s.io/kubernetes/cmd/kubeadm/app/util/system` module and is used everywhere instead of local constants.